new-site/yarn.lock

```diff
@@ -7188,10 +7188,10 @@ image-q@^1.1.1:
   resolved "https://registry.yarnpkg.com/image-q/-/image-q-1.1.1.tgz#fc84099664460b90ca862d9300b6bfbbbfbf8056"
   integrity sha1-/IQJlmRGC5DKhi2TALa/u7+/gFY=
 
-immer@8.0.1, immer@9.0.6:
```

Comment:
This reverts the immer crictical severity fix 😐

Reply:
But we might need to revert the Immer fix because it might have some  effect on hot reloading not working